src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Search.cshtml

```diff
@@ -30,14 +30,14 @@
         
         <form method="post" novalidate>
             <div class="govuk-form-group @(error?.FormGroupClass)" id="search-options">
-                <label class="govuk-label govuk-label--s" for="Postcode">
+                <label class="govuk-label govuk-label--s" for="postcode">
                     Enter a postcode
                 </label>
-                <div id="account-number-hint" class="govuk-hint">
+                <div id="postcode-services-hint" class="govuk-hint">
```

Comment:
I think technically this should just be `postcode-hint` but it's not like this doesn't work